internal/grpcwrapper/rawtopic/client.go

```diff
@@ -104,6 +104,7 @@ func (c *Client) StreamRead(ctxStreamLifeTime context.Context) (rawtopicreader.S
 func (c *Client) StreamWrite(
 	ctxStreamLifeTime context.Context,
 	tracer *trace.Topic,
+	logContext *context.Context,
```

Comment:
What about merge contextx before  call StreamWrite?

why do you need a pointer?

Reply:
merged before call, removed additional arg